plugins/reporters/fossid/src/main/resources/templates/asciidoc/fossid_snippet.ftl

```diff
@@ -76,12 +76,12 @@ License(s):
 |===
 | Source Location | pURL | License | File | URL | Score | Release Date
 
-.${snippetCount}+|
+.${snippetCount*2}+|
 [#if helper.isFullFileLocation(sourceLocation)]
 Full match
 [#else]
 Partial match +
-${sourceLocation.startLine}-${sourceLocation.endLine}
+${sourceLocation.startLine?c}-${sourceLocation.endLine?c}
```

Comment:
Out of curiosity, what does `?c` do?

Reply:
https://freemarker.apache.org/docs/ref_builtins_number.html#ref_builtin_c

It converts the number to a number "for the machine".
Without it, the number are rendered in the locale e.g. 1000->1.000